### ISSUE TITLE
Homebrew package

### ISSUE THREAD

**Issue by oknozor** [oknozor/unveil-rs#20]
This shall be easy : https://github.com/Homebrew/homebrew-core/blob/6ffa590119b42f1337584e28066e4733c5431e2a/Formula/mdbook.rb

**Comment by dnaka91** [comment]
Hello @oknozor I created a brew formula for this project, but the homebrew audit gave a few warnings, so it's unlikely that it will be accepted. Namely this two:

- Stable version URLs should not contain alpha
- GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)

If you like I can just simply try or you can create your own repository of formulas by running `brew new-tap` to create one from a template and then I can add the formula as a PR to that repo.

**Comment by oknozor** [comment]
Hey @dnaka91 Thanks a lot for your help ! 
I have just created a [homebrew-tap](https://github.com/oknozor/homebrew-tap) repo. Feel free to submit a PR.

**Comment by dnaka91** [comment]
Great thank you. I will add a slightly different Formula though to make installation easier.

When using a personal tap, the formulas will be installed from source by default as there is no automated system to create pre-compiled binaries. For users it means that brew would install the whole Rust toolchain and build unveil-rs from source.

I saw that your latest release contains a gzipped binary for MacOS as well as Linux so I will make a formula that uses these instead. In the end it just downloads the binary and puts it in the right place but for the end user it's more convenient this way. Most personal taps I saw so far do it this way as well.
